src/calendar-app/calendar/view/CalendarView.ts

```diff
@@ -190,7 +190,7 @@ export class CalendarView extends BaseTopLevelView implements TopLevelView<Calen
 									onEventClicked: (calendarEvent, domEvent) => this.onEventSelected(calendarEvent, domEvent, this.htmlSanitizer),
 									onEventKeyDown: this.handleEventKeyDown(),
 									onNewEvent: (date) => {
-										this.createNewEventDialog(getNextHalfHour())
+										this.createNewEventDialog(date)
```

Comment:
I think this should be the selected date but with the next half hour to keep the same behavior as requested  in #7126

Passing the date as parameter to getNextHalfHour should be enough.
```ts
export function getNextHalfHour(referenceDate?: Date): Date {
	let date: Date = referenceDate ?? new Date()
```